Scripts/Matlab/hopsan.m

```diff
@@ -0,0 +1,67 @@
+classdef hopsan
+    methods
+        function checkMessages(obj)
+            bufsize=1000;
+            buf=libpointer('string',blanks(bufsize));
+            status = 0;
+            while (status == 0)
+                [status,msg] = calllib('hopsanc','getMessage',buf,bufsize);    
+                disp(msg);
+            end
+        end
+        function obj = hopsan()
+            path = fullfile(getenv("HOPSANHOME"), '\bin\hopsanc.dll');
```

Comment:
What about Linux ? 
Also HOPSANHOME vs. HOPSAN_HOME

Reply:
Ok, I added Linux support, but I cannot test it since I don't have Matlab installed on Linux.

Reply:
Also need to load the default library on linux, or make the same solution as in python that load for windows also, but only if the file exists

Reply:
Ok, I added loading the default library. But again, it is untested on Linux. But it should work, since it works the same way as for Python.